lib/zenoss/events/event.rb

```diff
@@ -25,11 +26,25 @@ class Event < OpenStruct
       # Initialize this object from a Hash returned via the JSON api
       # @param[Zenoss] zenoss the current instance we are connecting with
       # @param[Hash] zhash a hash of values used to create this Event instance
+      # @return[Time] self.firstTime, self.lastTime
```

Comment:
It would be very unusual to call `#initialize` directly so it's probably best not to include this comment.